Example/ReloadlyAuthentication/Info.plist

```diff
@@ -4,6 +4,12 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
+	<key>ReloadlySDKClientId</key>
+	<string>NxhvFxDrXtX34VL1DLHsbQGLOPgE8qL9</string>
+	<key>ReloadlySDKSecretId</key>
+	<string>c7gcNR4dOIAlmalojV35okbq--tqqVhPPr2bvcsXlmKadewvaXdRSxENwfpyFywh</string>
```

Comment:
@markevychtaras Why are these credentials still being hard-coded?

Reply:
@epiard13 This should be OK since the users need to provide the creds in the Plist. Its up to the users to NOT checkin the Plist (via the .gitignore) file or employing any other ways to not make the production Plist public.

Reply:
Removed